app/widget/slider/sliderladder.cpp

```diff
@@ -291,6 +291,12 @@ void SliderLadderElement::UpdateLabel()
   } else {
     label_->setText(value_);
   }
+
+  // Increase the ladder width if displaying timecode
+  if (value_.contains(":")) {
+    label_->setFixedWidth(QtUtils::QFontMetricsWidth(label_->fontMetrics(), QStringLiteral("00:00:00:00")));
```

Comment:
Seems a little hacky to check the string for a colon. It might be better for the slider itself to provide the minimum width on the SliderLadder's construction, which can then be set on SliderLadderElement's construction.